core/templates/dev/head/pages/question_editor/QuestionEditorDirective.js

```diff
@@ -163,20 +163,23 @@ oppia.directive('questionEditor', [
             var state = $scope.question.getStateData();
             var contentIdsToAudioTranslations = (
               state.contentIdsToAudioTranslations);
+            var writtenTranslations = state.writtenTranslations;
             var updateQuestion = _updateQuestion;
             if (contentIdsToAudioTranslations.hasUnflaggedAudioTranslations(
               contentId)) {
               $uibModal.open({
                 templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
-                  '/components/forms/' +
-                  'mark_all_audio_as_needing_update_modal_directive.html'),
+                  '/components/forms/mark_all_audio_and_translations_as_' +
+                  'needing_update_modal_directive.html'),
                 backdrop: true,
-                resolve: {},
-                controller: 'MarkAllAudioAsNeedingUpdateController'
+                controller: (
+                  'MarkAllAudioAndTranslationsAsNeedingUpdateController')
```

Comment:
Shouldn't this controller be imported in topic_ediitor.html and skill_editor.html as well? As they use the question editor.

Reply:
I realized that we haven't added `MarkAllAudioAsNeedingUpdateController.js` in the skills and topic, I've added the modified files into proper places, thanks for noticing! (Although this wasn't required in the topic and skills as currently questions cannot have audio or text translations.)

Done!